### PR TITLE
bash: bump

### PR DESCRIPTION
**Pull request by kenmays**
Tested on Haiku R1B3.

### CODE REVIEW

File: app-shells/bash/bash-5.1.8.recipe [hunk 1]
```diff
@@ -63,7 +63,7 @@ PATCH()
 BUILD()
 {
 	CPPFLAGS="-DUSE_MKTEMP=1 -DUSE_MKSTEMP=1" runConfigure ./configure \
-		--without-bash-malloc --with-installed-readline
+		--without-bash-malloc --disable-rpath --enable-disabled-builtins --with-installed-readline
```
Comment (extrowerk) [inline app-shells/bash/bash-5.1.8.recipe:66]: What does the `--enable-disabled-builtins` do?

Comment (pulkomandy) [reply app-shells/bash/bash-5.1.8.recipe]: > --enable-disabled-builtins
> Allow builtin commands to be invoked via ‘builtin xxx’ even after xxx has been disabled using ‘enable -n xxx’. See Bash Builtins, for details of the builtin and enable builtin commands.

Sounds quite useful to me, why remove it?

**Comment by korli** [comment]
there is also a pending patch for readline: https://mirror.easyname.at/gnu/readline/readline-8.1-patches/readline81-001

**Comment by kenmays** [comment]
@korli: https://github.com/haikuports/haikuports/pull/6106

**Pull request closed without merging**